.github/workflows/main.yml

```
@@ -142,6 +142,6 @@ jobs:
               repo: "endless-key-app",
               event_type: "kolibri-explore-plugin-release",
               client_payload: {
-                VERSION: "$VERSION",
+                VERSION: "${{ env.VERSION }}",
```

I would guess without proof that this would work:

```suggestion
                VERSION: env.VERSION,
```

---

Or maybe not. Merging what you have!

---

I'm not really sure. I think that would imply that the substitution would happen within the action in node. I don't know if `env` actually corresponds to environment variables in that context. I think directly substituting from the context variable with the expression syntax is more likely to work, though.

---

@dbnicholson you got it!  Thanks!